extensions/qute/deployment/src/main/java/io/quarkus/qute/deployment/QuteDevModeProcessor.java

```diff
@@ -53,7 +53,7 @@ public boolean test(String t) {
                             return false;
                         }
                     });
-            try (ClassCreator classCreator = ClassCreator.builder().className("io.quarkus.qute.test.QuteDummyGlobals")
+            try (ClassCreator classCreator = ClassCreator.builder().className("org.acme.qute.test.QuteDummyGlobals")
```

Comment:
Why changing this package name?

Reply:
Yeah this is a bit odd.

Reply:
> Why changing this package name?

Because we need to simulate an ordering issue where the generated globals are sorted by the FQCN of the declaring class.